website/src/main/tut/docs/installation/core.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: docs
+title: scalafix-core
+---
+
+## scalafix-core
+Scalafix can be used as a library to run custom rules.
+
+```scala
+// ===> build.sbt
+libraryDependencies += "ch.epfl.scala" %% "scalafix-core" % "@V.stableVersion"
+// (optional) Scala.js is also supported
+libraryDependencies += "ch.epfl.scala" %%% "scalafix-core" % "@V.stableVersion"
+```
+
+Example usage of the syntactic API.
+
+```scala
+{% include MyRule.scala %}
+```
+
+```scala
```

Comment:
Can we use tut here for typechecking?

Reply:
Yes, I think so :) Let me try

Reply:
Actually, not so easy. `tut` passes before `Jekyll` so `MyRule` is not in scope